lib/elixir/lib/gen_server/loop.ex

```diff
@@ -0,0 +1,392 @@
+defmodule GenServer.Loop do
+  @moduledoc false
+
+  @typep gen_name() ::
+    pid() | {:local, atom()} | {:global, term()} | {:via, module(), term}
```

Comment:
Since we are implementing our own, should we stop supporting sending `{:local, atom()}`in?